### ISSUE TITLE
ESPnet install error on Ubuntu 20.04 with Docker in WSL2

### ISSUE THREAD

**Issue by Shin-ichi-Takayama** [espnet/espnet#4572]
**Describe the issue**
I am trying to install ESPnet using Docker on Ubuntu 20.04 with WSL2. 
However, I am getting errors along the way and it is not installing correctly.

**Show the `check_install.py` status by using the following command**
```
[x] python=3.7.13 (default, Mar 29 2022, 02:18:16)  [GCC 7.5.0]

Python modules:
[x] torch=1.10.1
[ ] torch cuda
[x] torch cudnn=8200
[x] torch nccl
[x] chainer=6.0.0
[ ] chainer cuda
[ ] chainer cudnn
[ ] cupy
[x] torchaudio=0.10.1
[x] torch_optimizer=0.3.0
[ ] warpctc_pytorch
[ ] warprnnt_pytorch
[ ] chainer_ctc
[ ] pyopenjtalk
[ ] tdmelodic_pyopenjtalk
[ ] kenlm
[ ] mmseg
[x] espnet=202207
[ ] fairseq
[ ] phonemizer
[ ] gtn
[ ] s3prl
[ ] transformers
[ ] speechbrain
[ ] k2
[ ] longformer
[ ] nlg-eval
[ ] datasets

Executables:
[ ] sclite
[ ] sph2pipe
[ ] PESQ
[ ] BeamformIt

INFO:
Use 'installers/install_warp-ctc.sh' to install warpctc_pytorch
Use 'installers/install_warp-transducer.sh' to install warprnnt_pytorch
Use 'installers/install_chainer_ctc.sh' to install chainer_ctc
Use 'installers/install_pyopenjtalk.sh' to install pyopenjtalk
Use 'installers/install_tdmelodic_pyopenjtalk.sh' to install tdmelodic_pyopenjtalk
Use 'installers/install_kenlm.sh' to install kenlm
Use 'installers/install_py3mmseg.sh' to install mmseg
Use 'installers/install_fairseq.sh' to install fairseq
Use 'installers/install_phonemizer.sh' to install phonemizer
Use 'installers/install_gtn.sh' to install gtn
Use 'installers/install_s3prl.sh' to install s3prl
Use 'installers/install_transformers.sh' to install transformers
Use 'installers/install_speechbrain.sh' to install speechbrain
Use 'installers/install_k2.sh' to install k2
Use 'installers/install_longformer.sh' to install longformer
Use 'installers/install_longformer.sh' to install nlg-eval
Use 'installers/install_longformer.sh' to install datasets
Use 'installers/install_sctk.sh' to install sclite
Use 'installers/install_sph2pipe.sh' to install sph2pipe
Use 'installers/install_pesq.sh' to install PESQ
Use 'installers/install_beamformit.sh' to install BeamformIt
```

**Basic environments:**
 - OS information: Ubuntu20.04
 - python version: 3.7.3
 - espnet version: 202207
 - Git hash 277ec3c33d2ca7f47d9d31c84e4dae54ce017bd7
   - Commit date:Wed Aug 10 13:32:09 2022 -0400
 - pytorch version 1.10.1
 
**Environments from `torch.utils.collect_env`:**
```
Collecting environment information...
PyTorch version: 1.10.1
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.4 LTS (x86_64)
GCC version: (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
Clang version: Could not collect
CMake version: version 3.16.3
Libc version: glibc-2.17

Python version: 3.7.13 (default, Mar 29 2022, 02:18:16)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-debian-bullseye-sid
Is CUDA available: True
CUDA runtime version: 11.3.109
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 3060
Nvidia driver version: 516.93
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.21.5
[pip3] pytorch-ranger==0.1.1
[pip3] pytorch-wpe==0.0.1
[pip3] torch==1.10.1
[pip3] torch-complex==0.4.3
[pip3] torch-optimizer==0.3.0
[pip3] torchaudio==0.10.1
[conda] blas                      1.0                         mkl
[conda] cudatoolkit               11.3.1               h2bc3f7f_2
[conda] mkl                       2021.4.0           h06a4308_640
[conda] mkl-service               2.4.0            py37h7f8727e_0
[conda] mkl_fft                   1.3.1            py37hd3c417c_0
[conda] mkl_random                1.2.2            py37h51133e4_0
[conda] numpy                     1.21.4                   pypi_0    pypi
[conda] numpy-base                1.21.5           py37ha15fc14_3
[conda] pytorch                   1.10.1          py3.7_cuda11.3_cudnn8.2.0_0    pytorch
[conda] pytorch-mutex             1.0                        cuda    pytorch
[conda] pytorch-ranger            0.1.1                    pypi_0    pypi
[conda] pytorch-wpe               0.0.1                    pypi_0    pypi
[conda] torch-complex             0.4.3                    pypi_0    pypi
[conda] torch-optimizer           0.3.0                    pypi_0    pypi
[conda] torchaudio                0.10.1               py37_cu113    pytorch
```

**To Reproduce**
Steps to reproduce the behavior by showing us the specific installation commands with their arguments, e.g.,
```
cd ~/espnet/tools
make TH_VERSION=1.10.1
```

Line 45-46 of the following Dockerfile.
```
FROM nvidia/cuda:11.3.0-devel-ubuntu20.04

#Proxy Settings
ENV http_proxy http://hoge.co.jp:8080
ENV https_proxy http://hoge.co.jp:8080
ENV PIP_DEFAULT_TIMEOUT=1000

#setting timezone
ENV TZ=Asia/Tokyo
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone

RUN mv /var/lib/apt/lists ~/

#set timeout for apt
RUN /bin/echo -e "Acquire::http::Timeout \"300\";\n\
Acquire::ftp::Timeout \"300\";" >> /etc/apt/apt.conf.d/99timeout

RUN apt update && apt install -y tzdata && \
	apt install -y build-essential && \
	apt install -y software-properties-common && \
	apt install -y cmake sox ffmpeg flac curl git man unzip vim wget sudo && \
	apt install -y python3 python3-pip python3-distutils && \
	apt install -y libfreetype6 libfreetype6-dev pkg-config

#install Anaconda
RUN \
	apt install libgl1-mesa-glx libegl1-mesa libxrandr2 libxrandr2 libxss1 libxcursor1 libxcomposite1 libasound2 libxi6 libxtst6 wget -y && \
	wget -P /opt https://repo.anaconda.com/archive/Anaconda3-2022.05-Linux-x86_64.sh && \
	bash /opt/Anaconda3-2022.05-Linux-x86_64.sh -b -p /opt/anaconda3 && \
	rm /opt/Anaconda3-2022.05-Linux-x86_64.sh && \
	echo "export PATH="/opt/anaconda3/bin:$PATH"" >> ~/.bashrc && \
	. ~/.bashrc && \
	conda config --set proxy_servers.http http://proxy.mei.co.jp:8080 && \
	conda config --set proxy_servers.https http://proxy.mei.co.jp:8080 && \
	conda config --set remote_read_timeout_secs 6000 && \
	conda config --show && \
	conda init

#install espnet
RUN \
	cd ~ && \
	git clone https://github.com/espnet/espnet && \
	cd ~/espnet/tools && \
	./setup_anaconda.sh anaconda espnet 3.8 && \
	cd ~/espnet/tools && \
	make TH_VERSION=1.10.1 && \
	cd ~/espnet/tools && \
	. ./activate_python.sh; python3 check_install.py
```

**Error logs**
Paste the error logs. If applicable, add screenshots to help explain your problem.
[install.log](https://github.com/espnet/espnet/files/9333372/install.log)

**Comment by b-flo** [comment]
Hi,

> I am trying to install ESPnet using Docker on Ubuntu 20.04 with WSL2.
> However, I am getting errors along the way and it is not installing correctly.

From what I see, there is no errors in the log and mandatory things were installed. Could you please point out the error and what was not installed correctly?

**Comment by Shin-ichi-Takayama** [comment]
@b-flo 
Thank you for your reply.

There are errors on lines 3499 and 3500. Is this a problem?
Also, is it ok that torch cuda is not checked? I have installed CUDA 11.3.

I was getting the error when I trained with the original data.
I thought these errors were the cause, but when I changed nj from 32 to 4, it worked fine.
It was the memory limitation of WSL2 that was the cause and not these errors.
Sorry for the trouble.
